src/prefect/deployments/flow_runs.py

```diff
@@ -156,6 +161,17 @@ async def run_deployment(
     else:
         parent_task_run_id = None
 
+    if flow_run_ctx and flow_run_ctx.flow_run:
+        traceparent = flow_run_ctx.flow_run.labels.get(LABELS_TRACEPARENT_KEY)
+    else:
+        traceparent = None
+
+    trace_labels = {}
+    if traceparent:
+        carrier: KeyValueLabels = {TRACEPARENT_KEY: traceparent}
+
+        trace_labels = {LABELS_TRACEPARENT_KEY: carrier[TRACEPARENT_KEY]}
```

Comment:
This looks like it can be simplified
```suggestion
    trace_labels = {LABELS_TRACEPARENT_KEY: traceparent} if traceparent else {}
```